generators/php/sdk/src/SdkGeneratorContext.ts

```diff
@@ -74,6 +74,12 @@ export class SdkGeneratorContext extends AbstractPhpGeneratorContext<SdkCustomCo
     }
 
     public getEndpointMethodName(endpoint: HttpEndpoint): string {
+        // TODO: Propogate reserved keywords through IR via CasingsGenerator.
+        const reservedKeywords = ["use", "clone", "list"];
```

Comment:
nit: Can we declare this in [constants.ts](https://github.com/fern-api/fern/blob/main/generators/php/sdk/src/constants.ts) as `RESERVED_METHOD_NAMES`?